packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -2,263 +2,282 @@ import { and, eq } from "drizzle-orm"
 import {
   int,
   timestamp,
-  mysqlTable as defaultMySqlTableFn,
   primaryKey,
   varchar,
-  MySqlTableFn,
   MySqlDatabase,
+  mysqlTable,
+  MySqlTableWithColumns,
+  TableConfig,
+  QueryResultHKT,
+  PreparedQueryHKTBase,
+  index,
 } from "drizzle-orm/mysql-core"
 
-import type { Adapter, AdapterAccount } from "@auth/core/adapters"
+import type {
+  Adapter,
+  AdapterUser,
+  AdapterAccount,
+  AdapterSession,
+  VerificationToken,
+} from "@auth/core/adapters"
 
-export function createTables(mySqlTable: MySqlTableFn) {
-  const users = mySqlTable("user", {
-    id: varchar("id", { length: 255 }).notNull().primaryKey(),
-    name: varchar("name", { length: 255 }),
-    email: varchar("email", { length: 255 }).notNull(),
-    emailVerified: timestamp("emailVerified", {
-      mode: "date",
-      fsp: 3,
-    }).defaultNow(),
-    image: varchar("image", { length: 255 }),
-  })
+import { randomUUID } from "crypto"
 
-  const accounts = mySqlTable(
-    "account",
-    {
-      userId: varchar("userId", { length: 255 })
-        .notNull()
-        .references(() => users.id, { onDelete: "cascade" }),
-      type: varchar("type", { length: 255 })
-        .$type<AdapterAccount["type"]>()
-        .notNull(),
-      provider: varchar("provider", { length: 255 }).notNull(),
-      providerAccountId: varchar("providerAccountId", {
-        length: 255,
-      }).notNull(),
-      refresh_token: varchar("refresh_token", { length: 255 }),
-      access_token: varchar("access_token", { length: 255 }),
-      expires_at: int("expires_at"),
-      token_type: varchar("token_type", { length: 255 }),
-      scope: varchar("scope", { length: 255 }),
-      id_token: varchar("id_token", { length: 255 }),
-      session_state: varchar("session_state", { length: 255 }),
-    },
-    (account) => ({
-      compoundKey: primaryKey(account.provider, account.providerAccountId),
-    })
-  )
+export const mysqlUsersTable = mysqlTable("user" as string, {
+  id: varchar("id", { length: 255 })
+    .primaryKey()
+    .$defaultFn(() => randomUUID()),
```

Comment:
so this just wont have any affect since the `createUser` function always generates an id with randomUUID()